### PR TITLE
Fix code formatting for aws s3 distribution

### PR DESCRIPTION
**Pull request by rute-santos**
### Your pull request should…

* [ ] have a concise title
* [ ] [close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists
* [ ] be against the correct branch (features can only go into develop)
* [ ] include migration scripts and documentation, if appropriate
* [ ] pass automated tests
* [ ] have a clean commit history
* [ ] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

### CODE REVIEW

File: modules/distribution-service-aws-s3/src/main/java/org/opencastproject/distribution/aws/s3/AwsS3DistributionServiceImpl.java [hunk 1]
```diff
@@ -559,9 +556,9 @@ public Job retract(String channelId, MediaPackage mediapackage, Set<String> elem
 
   @Override
   public List<MediaPackageElement> distributeSync(String channelId, MediaPackage mediapackage, Set<String> elementIds,
-         boolean checkAvailability) throws DistributionException {
-    final MediaPackageElement[] distributedElements =
-        distributeElements(channelId, mediapackage, elementIds, checkAvailability);
+          boolean checkAvailability) throws DistributionException {
+    final MediaPackageElement[] distributedElements = distributeElements(channelId, mediapackage, elementIds,
+            checkAvailability);
```
Comment (LukasKalbertodt) [inline modules/distribution-service-aws-s3/src/main/java/org/opencastproject/distribution/aws/s3/AwsS3DistributionServiceImpl.java:573]: Not a huge fan of this function call change, but it's ok.

File: modules/distribution-service-aws-s3/src/main/java/org/opencastproject/distribution/aws/s3/AwsS3DistributionServiceImpl.java [hunk 2]
```diff
@@ -291,12 +292,8 @@ public void activate(ComponentContext cc) {
...
 
       s3 = AmazonS3ClientBuilder.standard()
-              .withEndpointConfiguration(new AwsClientBuilder.EndpointConfiguration(endpoint
-                      , regionStr))
-              .withPathStyleAccessEnabled(pathStyle)
-              .withCredentials(provider)
-              .build();
-
+              .withEndpointConfiguration(new AwsClientBuilder.EndpointConfiguration(endpoint, regionStr))
+              .withPathStyleAccessEnabled(pathStyle).withCredentials(provider).build();
```
Comment (LukasKalbertodt) [inline modules/distribution-service-aws-s3/src/main/java/org/opencastproject/distribution/aws/s3/AwsS3DistributionServiceImpl.java:307]: The `withEndpointConfiguration` call is an improvement, but I'm not sure if I agree with combining the last three calls into one line. I don't really think it improves readability.

**Review by LukasKalbertodt — APPROVED** [review]
This looks like an overall improvement to me. Thanks!

I left two inline comments where I personally don't fully agree with the changes, but my opinion here isn't that important: I'm not used to code formatting in the Java world. So feel free to just ignore those comments.

Please note that I also touch this file in [this formatting PR](https://github.com/opencast/opencast/pull/2340). I change a couple other things as well, but your PR also changes a few things I don't. I would be fine with rebasing my PR onto this one, if this one is merged first.

**Comment by rute-santos** [comment]
Hi @LukasKalbertodt ,
Thank you for the review! Just fyi, I just used the code formatting that comes with Opencast, I did not create any new formatting rules :)

**Review by KatrinIhler — APPROVED** [review]
I guess you could nitpick this, but overall it's an improvement and I don't want to cause you more work since I was the one who demanded this PR in the first place. ;)

**Comment by KatrinIhler** [comment]
And also in general I wouldn't put formatting changes into a minor release, but since this is associated to the bug fix in #2318 and the changes are very minor, I think this is okay this time.

**Pull request merged**